modules/ownKeys.js

```diff
@@ -0,0 +1,19 @@
+import objectKeys from 'object-keys'
+
+const ownKeys = (object) => {
```

Comment:
Perhaps we could call this `ownEnumerableKeys` to avoid confusing it with `Reflect.ownKeys`?